### ISSUE TITLE
Feedback regarding multiple announcements on rust.cologne

### ISSUE THREAD

**Issue by colin-kiegel** [Rustaceans/rust-cologne#17]
@killercup You asked for feedback. :-)

I see two problems
1. The titles are very long
2. future meetups are not very visible

Suggestions:
* using ISO-international dates `YYYY-MM-DD`, i.e. `2016-11-16` instead of american `Nov. 16th, 2016`
* adding the timezone `19:00 CET`
* removing the location from the title to make it shorter [it's least important and currently alsways the same plus it's very redundant for the next meetup layout with directions and maps following the title]
* if possible move the date and time into a slightly smaller subtitle to make the titles even shorter, while retaining some focus on the dates
* Increse the fonts for 'more future meetups: ...' to match 'next meetup: ...'

What do you think about this?

**Comment by colin-kiegel** [comment]
PS: ISO-dateformat https://en.wikipedia.org/wiki/ISO_8601

**Comment by killercup** [comment]
Thanks, @colin-kiegel, this is great!

> using ISO-international dates YYYY-MM-DD, i.e. 2016-11-16 instead of american Nov. 16th, 2016

Good point

> adding the timezone 19:00 CET

I think it's obvious that timezone Cologne happens to be in is meant  ;)

(The HTML metadata on the even page includes the full 8601 representation, by the way.)

> removing the location from the title to make it shorter […]
> 
> if possible move the date and time into a slightly smaller subtitle to make the titles even shorter, while retaining some focus on the dates

While we know it's almost always the same location, this is not obvious to new people.

Have a look at how the title is shown on the archive page (I changed this recently):

![bildschirmfoto 2016-11-12 um 19 16 50](https://cloud.githubusercontent.com/assets/20063/20239947/c33c2cf4-a90c-11e6-93ea-5a77b5bae770.png)

This comes all back to:

> The titles are very long

When I first designed the web site (the aesthetics as well as the data structures), our meetups didn't have official titles ("First meetup, talks and drinks", "First 2016 Meetup", etc.), so it felt wrong to put an emphasis on those.

What remains is to basically just give people some coordinates in space-time.

> Increse the fonts for 'more future meetups: ...' to match 'next meetup: ...'

It is currently set to match 'Previous meetup'. Maybe they should all be in the style of the 'Featured Impressions' headline?

**Comment by killercup** [comment]
Or maybe, on the start page, we can just show it like this:

> ## Next Meetup: Open Source Audio Meetup
> 
> # Wednesday, Nov. 16th, 2016 19:00
> 
> Chaos Computer Club Cologne (c4) e.V.
> Heliosstr. 6a
> Köln

**Comment by colin-kiegel** [comment]
I think I like this most
![bildschirmfoto 2016-11-12 um 19 16 50](https://cloud.githubusercontent.com/assets/20063/20239947/c33c2cf4-a90c-11e6-93ea-5a77b5bae770.png)

But I would shorten the location to a hyperlink (like here http://colognejs.de/), e.g.:
- **at [C4](#)**

How about one of these as subtitle?
1. **Wednesday, 2016-11-16 19:00 at [C4](#)**
2. **Wednesday, Nov 16th, 2016 19:00 at [C4](#)**

We could also try to rearrange the layout of each meetup a bit:
- the location on the left is redundant with google maps and the (sub-)title (=> remove it)
- "this event was also announced on" could be moved downwards
- the whole description could be moved upwards (what we do is more important than where it was announced, right?)

**Comment by killercup** [comment]
Currently, the whole red line is a link, so I can't embed another link (to C4) in it. I also don't want to 'hide' the address in the map since not everyone will see it (it's slow to load on mobile and many ad block/privacy plugins filter Google embeds).

It would be very cool to reorganize these "event" components a bit, but we need to be careful to make it look good even when we don't have much data (e.g., no talks and no description).

I'll need to think about this some more. Maybe I get around to making a PR asking for feedback this weekend.

**Comment by colin-kiegel** [comment]
This is a mockup - partly inspired from the js cologne meetup and our current website. :-)
![image](https://cloud.githubusercontent.com/assets/7691186/20240717/a2f52dac-a91f-11e6-872c-25027879665b.png)

**Comment by xetra11** [comment]
I like it.
I always have a hard time understanding things on first glance. But this one is pretty good.

**Comment by killercup** [comment]
Looks good, @colin-kiegel!

I think I'd swap the styles for the meetup's title and the date/location and not use the same headline style for "Next Meetup" and "Directions" (maybe just omit the "Directions" headline?). I'll try to throw something together later and let you know how this works for events with less information.

**Comment by colin-kiegel** [comment]
Dropping the "Directions" headline sounds good. 

Regarding the title I tried different iterations myself and this is what I came up with. IMO the bold font-weight didn't look good at the datetime part at all - probably because it has so much punctuation.

Also I liked to have the main focus on the title, because while quickly glancing over the page it is easier to interpret than a bunch of numbers (easy to read == visually pleasing). I also liked the idea of giving the location a different visual clue and possibly link it somewhere.

And I am not worried about people missing the date. ;-)

This is just me explaining my reasoning. I am looking forward to whatever you come up with. :-)
